editor/state.js

```diff
@@ -303,6 +303,21 @@ export function isSidebarOpened( state = false, action ) {
 	return state;
 }
 
+export function editorWidth( state = null, action ) {
+	const layout = document.querySelector( '.editor-layout__content' );
```

Comment:
These seem like arguments that should be passed in through action properties, not extracted by the reducer itself.

>Given the same arguments, it should calculate the next state and return it. No surprises. No side effects. No API calls. No mutations. Just a calculation.

http://redux.js.org/docs/basics/Reducers.html#handling-actions